protocol-units/execution/opt-executor/src/executor/transaction_pipe.rs

```diff
@@ -46,22 +47,16 @@ impl Executor {
 									// Re-create the validator for each Tx because it uses a frozen version of the ledger.
 									let vm_validator = VMValidator::new(Arc::clone(&self.db.reader));
 									let tx_result = vm_validator.validate_transaction(transaction.clone())?;
-									// If the verification failed return the error status.
-									if let Some(vm_status) = tx_result.status() {
-										let ms = MempoolStatus::new(MempoolStatusCode::VmError);
-										let status: SubmissionStatus = (ms, Some(vm_status));
-										callback.send(Ok(status)).map_err(
-											|e| anyhow::anyhow!("Error sending callback: {:?}", e)
-										)?;
-										continue;
-									}
-
-									// add to the mempool
-									{
 
+									let status = if let Some(vm_status) = tx_result.status() {
```

Comment:
Blame @0xmovses's tab settings in `rustfmt.toml` for this diff gore.

On the other hand, this code could be moved out to a helper function to be less nested.

Reply:
True diff gore. We could try not using the rule `hard_tabs = true`.